src/iceberg_seals/search/predicting/predict_sealnet.py

```diff
@@ -45,8 +46,8 @@ def parse_args():
                                                        'used in subsequent steps of the pipeline')
     parser.add_argument('--models_folder', type=str, default='saved_models', help='folder where the model tar file is'
                                                                                   'saved')
-    parser.add_argument('--input_image', type=str, help='filename of input raster')
     parser.add_argument('--output_dir', type=str, help='folder where output shapefiles will be stored')
+    parser.add_argument('--save_heatmaps', type=int, help='boolean for saving heatmaps', default=0)
```

Comment:
@bspitzbart this value is an int, while the help message says boolean. In addition, it is passed to a boolean parameter later. I would suggest to change it to Boolean and set as the default `True`